fixtures/common.py

```diff
@@ -69,23 +69,24 @@ def session_indexing_decorator():
     when a module is loaded. To get around this, we patch the decorator function then
     reload the relevant modules via importlib.
     """
-    mock_indexer_func = Mock()
+    mock_indexer_func = Mock(original=[])
```

Comment:
Should be renamed to `mock_persist_func` or something like that (that also goes for the value returned in the `SimpleNamespace`)